### ISSUE TITLE
OpenAPI 3.0 validation does not check for valid $ref's in pathitems

### ISSUE THREAD

**Issue by devjack** [Mermade/swagger2openapi#23]
### Summary 
When validating an openapi 3.0 specification, the validation does not check for a valid `$ref` as part of a PathItem Object. As a result PathItem objects that use $ref's (but that cannot be resolved e.g. to missing files) are incorrectly passing validation.

There is an [explicit check for each child field name](https://github.com/Mermade/swagger2openapi/blob/d9810573b17f3a06cd37b4ced98e95efe1993381/validate.js#L379)

A valid [PathItem Object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#path-item-object) according to the spec may include a $ref.

Example snippet:

```
paths:
    /SomePath:
        $ref: "paths/something.yml#/paths/SomePath"
```

I would expect behaviour for valid spec to be one of:
 - An error saying that it cannot resolve $ref on line X
 - An error stating that "paths/info.yml" does not exist on line X
 - An error stating that path/~1SomePath is not a valid PathItem object (invalid JSON Reference)

_Or something else equally helpful_

### Checklist

- [x] Validation: I believe my source definition is intentionally invalid OpenAPI 3.0.x but the validator fails to complain

**Comment by MikeRalphson** [comment]
I've published v2.7.0 - validating (not using the `validateSync` method) now resolves external references, and the `testRunner` has been updated to use this mode. Please let me know if it now correctly validates your definition(s).

**Comment by devjack** [comment]
@MikeRalphson This worked a treat. Thank you!
